src/main/java/soot/baf/AbstractInstSwitch.java

```diff
@@ -0,0 +1,392 @@
+package soot.baf;
+
+/*-
+ * #%L
+ * Soot - a J*va Optimization Framework
+ * %%
+ * Copyright (C) 2021 Timothy Hoffman
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ * #L%
+ */
+
+/**
+ * @author Timothy Hoffman
+ */
+public abstract class AbstractInstSwitch<T> implements InstSwitch {
+
```

Comment:
This class is not used anywhere in the PR.

Reply:
I ended up not using it but decided to keep it in because all of the other *Switch interfaces have an Abstract implementation (which is what I based this on) and I thought it might be a useful framework class to have.

Reply:
when it is not used, please remove it